packages/apps/fortune/exchange-oracle/server/src/modules/webhook/webhook.service.spec.ts

```diff
@@ -306,6 +307,22 @@ describe('WebhookService', () => {
         ),
       ).rejects.toThrowError('Invalid outgoing event type');
     });
+
+    it('should throw NotFoundError if operator is not found', async () => {
+      (EscrowClient.build as any).mockImplementation(() => ({
+        getJobLauncherAddress: jest.fn().mockResolvedValue(MOCK_ADDRESS),
+      }));
+
+      (OperatorUtils.getOperator as any).mockResolvedValue(null);
+
+      await expect(
+        (webhookService as any).getOracleWebhookUrl(
+          JOB_LAUNCHER_WEBHOOK_URL,
+          ChainId.LOCALHOST,
+          EventType.ESCROW_FAILED,
+        ),
+      ).rejects.toThrow(NotFoundError);
```

Comment:
Better to check on error message just in case there are multiple `NotFoundError` thrown from the method